gui/preload-main.js

```diff
@@ -1944,6 +1959,9 @@ var speedChartOptions = {
         var dxCall = document.createElement("td");
         var dxCallText = document.createElement('span');
         dxCallText.innerText = arg.stations[i]['dxcallsign'];
+        row.addEventListener("click", function(e) {
+            document.getElementById("dxCall").value = dxCallText.innerText;
```

Comment:
Well done! We should also check if the event listener already exists. In case of updating the row, ir will be created twice or more. 
See preload-chat for example 🙂

Reply:
Is that necessary?  The table gets cleared than recreated on each tnc-state message (as far as I can tell).

Reply:
Not sure, I only have a small screen (mobile phone) . If it's working, we can keep if, but should keep it in mind in case if problems. Nevertheless a nice feature I was missing as well.

Reply:
Did some quick research and it sounds the engine with GC the events if it's owner is destroyed.  I think it'll be ok.